src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -686,6 +696,15 @@ export class ServerView extends DirMixin(PolymerElement) {
       this.accessKeySortDirection = 1;
       /** @type {(msgId: string, ...params: string[]) => string} */
       this.localize = null;
+      this.currentPage = '';
+      this.selectedTab = 'connections';
+    }
+
+    selectPage(page) {
+      if (page !== 'progressView') {
+        this.$.progressView.stop();
```

Comment:
Let's rename the methods to startAnimation and stopAnimation. It will make it clearer what's actually doing. I had to open the code to figure out what exactly was getting stopped.